test/journeys/specs/recents/dataApi/basic.js

```diff
@@ -186,5 +209,17 @@ describe('Widget Recents', () => {
         'does not show call button');
       });
     });
+
+    describe('incoming call', () => {
+      it('should display incoming call screen', () => {
+        browserRemote.element(meetElements.meetWidget).element(meetElements.callButton).waitForVisible();
+        browserRemote.element(meetElements.callButton).click();
+        browserLocal.waitUntil(() =>
+          browserLocal.element(`${elements.answerButton}`).isVisible(),
```

Comment:
Could this just be ` browserLocal.element(elements.answerButton).isVisible()`?

Reply:
copypasta, let me see where that was from and fix both